tests/xml/XMLBearTest.py

```diff
@@ -18,14 +20,59 @@
 <a>hey and hi</a>
 """.splitlines(keepends=True)
 
-InvalidXMLCorrectedTest = verify_local_bear(
-                                        XMLBear,
-                                        valid_files=(valid_xml_file,),
-                                        invalid_files=(invalid_xml_file,),
-                                        tempfile_kwargs={"suffix": ".xml"})
-
-InvalidXMLSyntaxTest = verify_local_bear(
-                                        XMLBear,
-                                        valid_files=(valid_xml_chars,),
-                                        invalid_files=(invalid_xml_chars,),
-                                        tempfile_kwargs={"suffix": ".xml"})
+dtd_file = os.path.join(os.path.dirname(__file__),
+                        "test_files",
+                        "note.dtd")
+
+schema_file = os.path.join(os.path.dirname(__file__),
+                           "test_files",
+                           "note.xsd")
+
+valid_xml_path = list(open(os.path.join(os.path.dirname(__file__),
```

Comment:
I think you misunderstood me. Why not do this like how `invalid_xml_chars` is done ?

i.e. do something like:

```
valid_xml_path = """
<?xml version="1.0" encoding="UTF-8"?>
<note
 xmlns="http://www.w3schools.com"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:noNamespaceSchemaLocation="./note.xsd">
   <to>Tove</to>
   <from>Jani</from>
   <heading>Reminder</heading>
   <body>Don't forget me this weekend!</body>
</note>
"""
```

This makes it easier to find the xml file rather than opening a new file.

Note: Same for all the other files given in `valid_files` and `invalid_files`